### ISSUE TITLE
Changing iteration number in phyloseq::ordinate NMDS with unifrac distances

### ISSUE THREAD

**Issue by AllenLarocque** [joey711/phyloseq#1425]
Hi Phyloseq gang, 
Thanks for an amazing package! You are all amazing.

Quick question: I'm getting an error trying to change the iteration number in phyloseq::ordinate with NMDS and unifrac distance. Am I doing something stupid? Not sure exactly which function is handling the unifrac call - how to increase the iterations similar to using 'try = XXX'?.

Example:
`  ordBC = ordinate(GlobalPatterns, method = "NMDS", distance = "bray",try=100)`

Works great with Bray distance.

`ordUF` = ordinate(GlobalPatterns, method = "NMDS", distance = "wunifrac", try=100)`

Gives error. `Error in UniFrac(physeq, ...) : unused argument (try = 100)`

Note that it works a-ok with the default settings (20 runs):

`ordUF = ordinate(GlobalPatterns, method = "NMDS", distance = "wunifrac")`

I presume this is due to which function is handling the wunifrac distance call, but it was hard to parse this from the help file!

Cheers,
Allen

**Comment by RandolphQuek** [comment]
Hello all,

Bumping this up because I too, am trying to navigate this error.

Allen, did you manage to find a solution?

Cheerio, 

Randolph.

**Comment by AllenLarocque** [comment]
Hi Randolph,
Sadly I have not! I hope the devs get a chance to fix this - I presume it's just a little tweak somewhere (and I lack the skills necessary to do it myself!).
Cheers,
Allen

**Comment by ZLouisson** [comment]
Hi @AllenLarocque, did you end up sorting a work-around for this? Trying to use either trymax or previous.best arguments with unifrac but getting the same error. Cheers
